packages/build/package.json

```diff
@@ -24,19 +24,38 @@
   },
   "homepage": "https://github.com/Polymer/polymer-build#readme",
   "dependencies": {
-    "@babel/core": "^7.0.0-beta.42",
-    "@babel/plugin-external-helpers": "^7.0.0-beta.42",
-    "@babel/plugin-proposal-async-generator-functions": "^7.0.0-beta.42",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.42",
-    "@babel/plugin-syntax-async-generators": "^7.0.0-beta.42",
+    "@babel/core": "^7.0.0-beta.46",
+    "@babel/plugin-external-helpers": "^7.0.0-beta.46",
+    "@babel/plugin-proposal-async-generator-functions": "^7.0.0-beta.46",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.46",
+    "@babel/plugin-syntax-async-generators": "^7.0.0-beta.46",
     "@babel/plugin-syntax-dynamic-import": "^7.0.0-beta.43",
```

Comment:
I'm going to bump this one to 46 also, just to keep everything else at the same version.